elixir/apps/web/lib/web/live/settings/directory_sync.ex

```diff
@@ -285,35 +285,18 @@ defmodule Web.Settings.DirectorySync do
     if stored_token && Plug.Crypto.secure_compare(stored_token, state_token) do
       send(pid, :success)
 
-      # After admin consent, verify we can actually access the directory using client credentials
```

Comment:
This was redundant - we now do the Entra verification in the verification live view so it won't send back a success message to the listening view if not successful, and will print it there.